src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -63,6 +69,10 @@ def __init__(
                 "s3://example-bucket/some-key-prefix/". Objects will be saved in
                 a unique sub-directory of the specified location. If None, a
                 default data location will be used.
+            enable_network_isolation (bool): Specifies whether container will
+                run in network isolation mode. Network isolation mode restricts
+                the container access to outside networks (such as the internet).
+                Also known as internet-free mode (default: `False`).
```

Comment:
nit: Sphinx requires two backticks for `False`

Reply:
Done